roofit/roofitcore/src/RooHist.cxx

```diff
@@ -260,6 +261,65 @@ RooHist::RooHist(const RooHist& hist1, const RooHist& hist2, Double_t wgt1, Doub
 }
 
 
+RooHist::RooHist(const RooAbsReal &f, RooAbsRealLValue &x, Double_t xErrorFrac, Double_t scaleFactor, const RooArgSet *normVars, const RooFitResult* fr) :
```

Comment:
A doxygen-readable comment would be nice, such as:
```
/// Constructor with ...
/// 
/// More details.
/// \param[in] f The variable to be plotted.
/// \param[in] x ...
```

Reply:
Will be done.